src/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ target_include_directories(objxgboost
 target_compile_options(objxgboost
   PRIVATE
   $<$<AND:$<CXX_COMPILER_ID:MSVC>,$<COMPILE_LANGUAGE:CXX>>:/MP>
-  $<$<COMPILE_LANGUAGE:CXX>:-funroll-loops>)
+  $<$<AND:$<NOT:$<CXX_COMPILER_ID:MSVC>>,$<COMPILE_LANGUAGE:CXX>>:-funroll-loops>)
```

Comment:
Actually I was quite confused about this flag.  Isn't it determined by compiler automatically?

Reply:
No, not necessarily. GCC doesn't enable it for `-O3`: https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html